app/src/Components/Form.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+const initialForm = {
+  name: "",
+  email: "",
+  role: ""
+}
+
```

Comment:
I like the way you placed your state in different components. It's clear and makes sense.

Reply:
Thank you.